### ISSUE TITLE
Re-implement high-definition Gel Code

### ISSUE THREAD

**Issue by creative-collective** [ChaosInitiative/Portal-2-Community-Edition#103]
HD blobs that obey projected texture lighting exist, commented out within Portal 2 code. They were replaced with a different gel model because of console downgrades. I would like if you could locate and restore these blobs.

**Comment by CitadelCore** [comment]
We should be able to do that :)

**Comment by SpyceTewan** [comment]
Apparently there's cvars for this?

**Comment by creative-collective** [comment]
> 
> 
> Apparently there's cvars for this?

No, that makes the blobs _smoother_ They still don't obey projected texture shadows in that case.

**Comment by AddMGH** [comment]
This already works with the paintgun, not sure why/how, might need more looking into.

**Comment by CitadelCore** [comment]
According to what I can see shadows on blobs should only be disabled if gpu_level < 2, which seems to be working correctly. Can someone verify that shadows are still broken and post the value of their gpu_level cvar?

**Comment by ShygalCoco** [comment]
still not fixed
![image](https://user-images.githubusercontent.com/69723642/178327983-552d6b9e-69d2-45c2-a8f8-6b8f1aa686cc.png)
![image](https://user-images.githubusercontent.com/69723642/178328028-6de6044f-875d-4965-824e-cec2242ba3cf.png)
![image](https://user-images.githubusercontent.com/69723642/178328127-0ab357f4-2108-4a3c-9e05-36dff4f092e1.png)

**Comment by Trico-Everfire** [comment]
Still not fixed.

**Comment by ShygalCoco** [comment]
still not fixed (9/10/2022)
![image](https://user-images.githubusercontent.com/69723642/189503230-e357d9e6-6e78-444a-aa21-67c5c827991d.png)
![image](https://user-images.githubusercontent.com/69723642/189503236-de5e47e9-678a-4f8d-8ac0-6f69d7b13a73.png)
![image](https://user-images.githubusercontent.com/69723642/189503241-4d0712a5-15bf-4d7a-a301-dfc7a651585f.png)

**Comment by Trico-Everfire** [comment]
Still do not cast shadows.

**Comment by lenship2** [comment]
This is fixed through the addition of clustered rendering on projected textures
